ChatQnA/README.md

```diff
@@ -344,3 +344,22 @@ OPEA microservice deployment can easily be monitored through Grafana dashboards
 
 ![chatqna dashboards](./assets/img/chatqna_dashboards.png)
 ![tgi dashboard](./assets/img/tgi_dashboard.png)
+
+## Tracing Services with OpenTelemetry Tracing and Jaeger
+
+> NOTE: limited support. Only TGI serving on Gaudi is enabled for this feature.
+
+OPEA microservice and TGI/TEI serving can easily be traced through Jaeger dashboards in conjunction with OpenTelemetry Tracing feature. Follow the [README](https://github.com/opea-project/GenAIComps/tree/main/comps/cores/telemetry#tracing) to trace additional functions if needed.
+
+Tracing data is exported to http://{EXTERNAL_IP}:4318/v1/traces via Jaeger.
+Users could also get the external IP via below command.
+
+```bash
+ip route get 8.8.8.8 | grep -oP 'src \K[^ ]+'
```

Comment:
hostname -I | awk '{print $1}' does not work here, right?

Reply:
it won't tell you which one is the external IP. 
for example, you got that many IPs in one machine below, but you couldn't say that the first one is the external one.
![image](https://github.com/user-attachments/assets/9d26fc59-0305-42e4-a4d1-9cfe8f64aaf0)